docs/source/dev/analytics/jupyter.rst

```diff
@@ -3,83 +3,35 @@
 Jupyter Notebook
 ================
 
-Jupyter is a development tool that allows users to create notebooks containing comments and code, like iPython Notebook. Jupyter supports other languages via the use of 'kernels'.
+Jupyter is a web application that allows users to create and share documents containing live code,
+equations, visualizations and narrative text.  This section describes how to configure a Jupyter Python 3 notebook
+to allow access to Koverse data sets.
 
-To use Jupyter with Koverse and PySpark, first create a kernel.json file in a folder called 'koverse'
+The prerequisites for accessing Koverse data sets in a Jupyter Python 3 notebook are::
 
-Configure the kernel.json file as follows by setting the right value for SPARK_HOME::
+  Spark 1.6 installed as /opt/spark
+  Python 3.5
+  Jupyter
 
- {
-  "display_name": "Koverse PySpark",
-  "language": "python",
-  "argv": [
-   "/usr/bin/python",
-   "-m",
-   "ipykernel",
-   "-f",
-   "{connection_file}"
-  ],
-  "env": {
-   "SPARK_HOME": "",
-   "PYTHONPATH": “$SPARK_HOME/python/:$SPARK_HOME/python/lib/py4j-0.8.2.1-src.zip",
-   "PYTHONSTARTUP": “$SPARK_HOME/bin/pyspark",
-   "PYSPARK_SUBMIT_ARGS": "--deploy-mode client --jars koverse-sdk.jar,koverse-sdk-xml.jar,koverse-thrift.jar,koverse-server-base.jar,koverse-shaded-deps.jar,accumulo-core.jar,accumulo-fate.jar,accumulo-trace.jar,accumulo-tracer.jar,guava.jar,commons-validator-1.4.0.jar pyspark-shell"
-  }
- }
+Once the prerequisites are met, download the Koverse Spark datasource jar from::
 
+ https://s3.amazonaws.com/share.koverse.com/merck/koverse-spark-datasource-2.5.3-SNAPSHOT.jar
```

Comment:
Instead of linking to this specific version, we could change this to say that the user should download the version of the koverse-spark-datasource JAR file that matches their installed Koverse from https://nexus.koverse.com/nexus/content/groups/public/com/koverse/koverse-spark-datasource/. We could also say that they should contact support for the koverse-spark-datasource JAR file as we are going to have a relationship with anyone who needs it and we might want be involved given the issues we had with the 2.5.3 non-SNAPSHOT JAR. Whatever you think is best.